src/codewhisperer/util/editorContext.ts

```diff
@@ -219,15 +219,15 @@ function logSupplementalContext(supplementalContext: CodeWhispererSupplementalCo
         return
     }
 
-    getLogger().verbose(`
+    getLogger().debug(`CodeWhispererSupplementalContext:
             isUtg: ${supplementalContext.isUtg},
             isProcessTimeout: ${supplementalContext.isProcessTimeout},
             contentsLength: ${supplementalContext.contentsLength},
             latency: ${supplementalContext.latency},
         `)
 
     supplementalContext.supplementalContextItems.forEach((context, index) => {
-        getLogger().verbose(`
+        getLogger().debug(`
```

Comment:
Logging related messages in separate logger calls should not be done in a loop. Build _one_ string, then log that single string.

Reply:
understood